### DISCUSSION TITLE
InitializeAugmentation in v24 onwards

### DISCUSSION THREAD

**Discussion by KamilPacanek** [Yvand/EntraCP#271] [General]
Hi, why the body of this method was removed upon v24 release?

https://github.com/Yvand/EntraCP/blob/c794d9784f0fc6eda0ae77b244734bf9a7689ba5/Yvand.EntraCP/Yvand.EntraClaimsProvider/Configuration/ClaimsProviderConstants.cs#L312-L314

**Comment by Yvand** [comment]
Hi, this is because the tests in this method were redundant with similar tests already done earlier

**Reply by KamilPacanek** [reply]
Thanks!
